stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -6873,6 +6876,12 @@ pub mod test {
                 mode,
                 origin,
                 &ft_transfer_2,
+                Txid(
+                    "1232121232121232121232121232121232121232121232121232121232121232"
```

Comment:
I think this string is too long. Try 32 chars: `"12321212321212321212321212321212"`

Reply:
Or even simpler, `Txid([0;32])`.